docs/remote/wsl.md

```diff
@@ -271,12 +271,12 @@ Once the server is up and running the proxy settings on the *Remote* tab are use
 
 Extensions are typically designed and tested to either run locally or remotely, not both. However, if an extension supports it, you can force it to run in a particular location in your `settings.json` file.
 
-For example, the setting below will force the Docker extension to run locally and Debugger for Chrome extension to run remotely instead of their defaults:
+For example, the setting below will force the Docker extension to run locally and the Rust extension to run remotely instead of their defaults:
 
 ```json
 "remote.extensionKind": {
     "ms-azuretools.vscode-docker": [ "ui" ],
-    "msjsdiag.debugger-for-chrome": [ "workspace" ]
+    "rust-lang.rust-analyzer": [ "workspace" ]
```

Comment:
I'd guess this is already a workspace extension by default, so it might not be a good example.

Reply:
Any idea for a better default UI extension example?

Reply:
remote-ssh-edit? https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh-edit